emcache/node.py

```diff
@@ -17,6 +17,9 @@ class MemcachedHostAddress:
     address: str
     port: int
 
+    username: Optional[str] = None
```

Comment:
Initially I wanted to create authorization for each memcached instance, theoretically you can create 3 instances and each will have its own SASL-based credential information.

Reply:
remove